lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -151,8 +151,11 @@ static public function addGlobalIgnoredName($name)
      * Constructor.
      *
      * Initializes a new AnnotationReader.
+     *
+     * @param DocParser $parser
+     * @throws AnnotationException
```

Comment:
we don't put ``@throws`` everywhere (especially when it is not really meant to be catched by anything else than a global error page)